File: Platform/codesignal/Intro/8.matrixElementsSum/Solution.py
def solution(m):
  result = 0
  for col in range(len(m[0])):
    for row in range(len(m)):
      val = m[row][col]
      if val == 0:
        break
      else:
        result += val
  return result
        
  
print(solution([[0,1,1,2], 
                [0,5,0,0], 
                [2,0,3,3]])) # 9

File: Platform/codesignal/Intro/8.matrixElementsSum/Solution.js
function solution(m) {
  let sum = 0;
  for (let c=0; c < m[0].length; c++) {
    for (let r=0; r < m.length; r++){
      if (m[r][c] === 0){
        break;
      } else {
        sum += m[r][c];
      }
    }
  }
  return sum;
}

console.log(solution([[0, 1, 1, 2], 
                      [0, 5, 0, 0], 
                      [2, 0, 3, 3]]));